src/commands/buildCommands/trn.js

```diff
@@ -78,7 +78,7 @@ const componentTrnDefinitions$ = localTrns$.map(trnsFromFile => ({
 const fpLocale = lang =>
 	require(require.resolve(`flatpickr/dist/l10n/${lang}`, {
 		paths: [paths.repoRoot],
-	}))[lang];
+	}));
```

Comment:
Is there are way to log an error or add a unit test of some sort to prevent silent failures like this?